css/style.css

```diff
@@ -0,0 +1,348 @@
+@font-face {
+  font-family: "Roboto";
+  src: url("Roboto-Regular.ttf");
+}
+@font-face {
+  font-family: "Bungee Shade";
+  src: url("BungeeShade-Regular.ttf");
+}
+
+/* Base reset */
+:root {
+  --primary: #fad0d7;
+  --secondary: #ecd3dc;
+  --tertiary: #fff3f7;
+  --highlight: #c0fff4;
+  --white: #ffff;
+  --grey-dark: #565656;
+  --box: #7fffd4;
+}
+
+/* Global reset */
+* {
+  box-sizing: border-box;
+  margin: 0px;
+  padding: 0px;
+  text-decoration: none;
+}
+
+body {
+  font-size: 0.9em;
+  font-weight: 300;
+  font-family: "Roboto", sans-serif;
+  color: var(--grey-dark);
+}
+
+.company_name {
+  display: grid;
+  place-items: center;
+  background-image: url("../img/11b.jpg");
+  background-repeat: no-repeat;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+}
```

Comment:
Try adding `background-position: bottom` or `background-position: center` to keep the cupcakes more in view 🍰 It still doesn't look 100% right... but is nearly what you want I think 😅